apps/dotcom/src/components/SnapshotsEditor.tsx

```diff
@@ -72,7 +72,7 @@ export function SnapshotsEditor(props: SnapshotEditorProps) {
 	const handleUiEvent = useHandleUiEvents()
 	const sharingUiOverrides = useSharing()
 	const fileSystemUiOverrides = useFileSystem({ isMultiplayer: true })
-	const storeResult = useLocalStore(props.records, props.schema)
+	const storeResult = useInMemoryStore(props.records, props.schema)
```

Comment:
i don't know if we really need this hook, we can probably just pass the prop `snapshot={{ store: Object.fromEntries(...), schema: props.schema }}` and if it fails to load it should still show a good error screen?